tests/functional/wdio.conf.js

```diff
@@ -45,14 +45,15 @@ export const config = {
   // and 30 processes will get spawned. The property handles how many capabilities
   // from the same test should run tests.
   //
-  maxInstances: 2,
+  maxInstances: parseInt(process.env.EQ_FUNCTIONAL_TEST_MAX_INSTANCES) || 2,
```

Comment:
Currently, it makes the env mandatory. This way, it is not mandatory.
```suggestion
  maxInstances: parseInt(process.env.EQ_FUNCTIONAL_TEST_MAX_INSTANCES || 2),
```